src/components/conversations/ParticipantsView.tsx

```diff
@@ -25,23 +50,58 @@ const ParticipantsView: React.FC<ParticipantsViewProps> = (
     );
   }
 
+  const theme = useTheme();
+  // Display only a limited number of participant avatars
+  // Hide the rest behind a text element with a tooltip
+  const maxDisplayedParticipants =
+    props.maxDisplayedParticipants ?? DEFAULT_MAX_DISPLAYED_PARTICIPANTS;
+
+  const displayedParticipants = [];
+  const hiddenParticipants = [];
+
+  for (let i = 0; i < props.participants.length; i++) {
+    const participant = props.participants[i];
+    const name = fetchName(participant);
+    if (i < maxDisplayedParticipants) {
+      displayedParticipants.push(name);
+    } else {
+      hiddenParticipants.push(name);
+    }
+
+    if (i == MAX_HIDDEN_PARTICIPANTS - 1) {
```

Comment:
```
if (hiddenParticipants.length == MAX_HIDDEN_PARTICIPANTS) {
```

Reply:
or rename `MAX_HIDDEN_PARTICIPANTS` -> `MAX_ALL_PARTICIPANTS` depending on what you mean here

Reply:
fixed ✅